specs/eventing/control-plane.md

```diff
@@ -0,0 +1,1013 @@
+error to poorly-behaved clients).
+
+This document does not define the
+[data plane event delivery contract](./data-plane.md) (though it does describe
+how event delivery is configured). This document also does not prescribe
+specific implementations of supporting services such as access control,
+observability, or resource management.
+
+## Background
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" are to be
+interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+There is no formal specification of the Kubernetes API and Resource Model. This
+document assumes Kubernetes 1.21 behavior; this behavior will typically be
+supported by many future Kubernetes versions. Additionally, this document
+references specific core Kubernetes resources; these references can be
+illustrative (i.e. _an implementation on Kubernetes_) or descriptive (i.e. _this
+Kubernetes resource MUST be exposed_). References to these core Kubernetes
+resources will be annotated as either illustrative or descriptive.
+
+This document considers two users of a given Knative Eventing environment, and
+is particularly concerned with the expectations of developers (and language and
+tooling developers, by extension) deploying applications to the environment.
+
+- **Developers** configure Knative resources to implement an event-routing
+  architecture.
+- **Operators** (also known as **platform providers**) provision the underlying
+  event routing resources and manage the software configuration of Knative
+  Eventing and the underlying abstractions.
+
+## Resource Overview
+
+The Knative Eventing API provides a set of primitives to support both
+point-to-point communication channels (`messaging.knative.dev`) and
+content-based event routing (`eventing.knative.dev`). This specification
+describes API interfaces of Knative Eventing resources as well as the supported
+[event routing](./data-plane.md) logic and configuration settings.
+
+At the moment, the Knative Eventing specification does not contemplate any
+non-Kubernetes-backed implementations, and therefore does not specifically
+define the mapping of Kubernetes verbs (read, watch, patch, etc) to developer
+roles. See the [Overview documentation](./overview.md) for general definitions
+of the different API objects.
+
+## RBAC Profile
+
+In order to validate the controls described in
+[Resource Overview](#resource-overview), the following Kubernetes RBAC profile
+can be applied in a Kubernetes cluster. This Kubernetes RBAC is an _illustrative
+example_ of the minimal profile rather than a requirement. This Role is
+sufficient to develop, deploy, and manage event routing for an application
+within a single namespace. Knative Conformance tests against "MUST", "MUST NOT",
+"SHALL", "SHALL NOT", and "REQUIRED" conditions are expected to pass when using
```

Comment:
Kubernetes RBAC is not a requirement for Knative Eventing. The conformance requirement is that there is a cluster and a user set up (with a KUBECONFIG file for auth) which the tests execute aginst.

It would be possible (for example) to have a conformant cluster which implemented _no_ access control, only resource validation. On the other hand, a cluster where Channel / Subscription are forbidden to all users can't pass the conformance tests and so isn't conformant, even if it's running code which might otherwise be conformant.